### ISSUE TITLE
Unable to load GoogleCharts with Zone.js 0.8.8

### ISSUE THREAD

**Issue by Eddman** [angular/zone.js#753]
After upgrading to Zone.js 0.8.8 I'm unable to load GoogleCharts library. It is working fine in 0.8.7 so I assume something went wrong in 0.8.8.

Here a simple example of the loader:
`index.html:`
```html
<script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
```
`app.ts:`
```javascript
google.charts.load('current', {'packages': ['corechart', 'gauge']});
google.charts.setOnLoadCallback(() => { ... });
```

**Comment by JiaLiPassion** [comment]
What is the exception?

**Comment by Eddman** [comment]
That's the funny part: there is no exception at all. It creates a script element to load `https://www.gstatic.com/charts/45.1/loader.js` and appends `onload` callback:
```javascript
google.a.c.zb = function(a, c, d) {
        google.a.c.log("loadScript: " + a);
        var e = c.createElement("script");
        e.type = Ya;
        e.language = "javascript";
        e.async = !1;
        e.defer = !1;
        c = c.body || c.head || c.getElementsByTagName("HEAD").item(0) || c.documentElement;
        c.insertBefore(e, c.lastChild);
        d && google.a.c.md(e, d); <-- HERE
        e.src = a;
        google.a.c.log("end loadScript: " + a)
    }
```
```javascript
google.a.c.md = function(a, c) {
        if (c)
            if ("undefined" === typeof a.onload) {
                var d = !1;
                a.onreadystatechange = function() {
                    d || (a.readyState && a.readyState !== K ? google.a.c.window.setTimeout(a.onreadystatechange, 100) : (d = !0,
                    a.onreadystatechange = Y(),
                    google.a.c.window.setTimeout(c, 0)))
                }
            } else
                a.onload = c <-- HERE (a is the script element, c is the callback)
    }
```
But the `c` above is never called....

**Comment by Eddman** [comment]
OK, found it:
```javascript
var value = originalDescGet.apply(this); <-- Value is null here
value = desc.set.apply(this, [value]); <-- Setter does not return any value (as far as I know) so value is changed to undefined
if (typeof target['removeAttribute'] === 'function') {
    target.removeAttribute(prop);
}
return value; <-- Here it is undefined, but should be null
```
And in GoogleCharts:
```javascript
google.a.c.md = function(a, c) {
        if (c)
            if ("undefined" === typeof a.onload) { <-- Here the check obviously fails....and things do not work anymore
                var d = !1;
                a.onreadystatechange = function() {
                    d || (a.readyState && a.readyState !== K ? google.a.c.window.setTimeout(a.onreadystatechange, 100) : (d = !0,
                    a.onreadystatechange = Y(),
                    google.a.c.window.setTimeout(c, 0)))
                }
            } else
                a.onload = c
    }
```

**Comment by JiaLiPassion** [comment]
ok, thanks, it has been fixed in #747.

**Comment by Eddman** [comment]
Thanks! Just for info: when is the release planed? Thanks.

**Comment by JiaLiPassion** [comment]
@Eddman, I am not sure the release date, it should be soon!

**Comment by JiaLiPassion** [comment]
Has been fixed in 0.8.9, thanks.
